lib/diagnostic.js

```diff
@@ -17,7 +17,13 @@
 // Methods: sell, call
 
 // Answer Starts Here
-//
+Entity: User
+Attributes: name
+Methods: createEvent, deleteEvent
+
+Entity: Event
+Attributes: description, startTime, location, duration
+Methods: alert, remind, repeat
```

Comment:
Good choices for these. As a thought experiment, see if you can figure out how you might actually implement a `repeat` method!